tests/aurora/2dafile.cpp

```diff
@@ -0,0 +1,709 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Unit tests for our TwoDAFile class.
+ */
+
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "src/common/ustring.h"
+#include "src/common/error.h"
+#include "src/common/memreadstream.h"
+#include "src/common/memwritestream.h"
+
+#include "src/aurora/types.h"
+#include "src/aurora/2dafile.h"
+#include "src/aurora/gdafile.h"
+
+static const byte k2DAASCII[] =
```

Comment:
Should be `char`?

Yes, I know it ends up in a `MemoryReadStream`, but it seems odd to store text as byte here.